index.html

```diff
@@ -246,15 +262,37 @@ <h2>Get started</h2>
                 <p id="form-email" class="form-details">Your email</p>
                 <p id="form-text" class="form-details text-area-text">Enter your message here...</p>
             </div>
-            <form action="https://formspree.io/f/xzboarvz" method="post">
+            <form data-form id="contact-form" action="https://formspree.io/f/xzboarvz" method="post">
                 <ul>
-                    <li><input type="text" required maxlength="30" id="list-name"></li>
+                    <li><input type="text" required minlength="3" maxlength="30" id="list-name"></li>
                     <li><input type="email" required maxlength="50" id="list-email"></li>
                     <li><textarea name="textarea" cols="30" rows="10" required maxlength="500" id="list-textarea"></textarea></li>
                     <li><button type="submit" class="btn-get-started">Start collaboration</button></li>
                 </ul>
             </form>
         </div>
+        <div class="modal" id="modal">
+            <div class="modal-header">
+                <div data-title class="title"></div>
+                <button data-close-button class="btn-close" type="button">&times;</button>
+            </div>
+            <div class="modal-body">
+                <img src="#" alt="Project image" class="rectangle-34" id="image-1">
+                <img src="#" alt="Project image" class="rectangle-39" id="image-2">
+                <img src="#" alt="Project image" class="rectangle-40" id="image-3">
+                <img src="#" alt="Project image" class="rectangle-41" id="image-4">
+                <img src="#" alt="Project image" class="rectangle-42" id="image-5">
+                <div data-html-css class="html-css html-border"></div>
+                <div data-ruby-rails class="ruby-rails ruby-border"></div>
+                <div data-javascript class="javascript javascript-border"></div>
```

Comment:
- [ ] Kindly the error message should show up close to the "submit" button. when a validation error occurs.